CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - Upgrade github-pages in docs/Gemfile to resolve CVE-2021-28834 in kramdown dependency [cyberark/conjur#2099](https://github.com/cyberark/conjur/issues/2099)
 
+### Added
+- Rake task for password strength validation. Can be called with `rake password:validate_strength[password]`
```

Comment:
do we need quotes around `password` when we run this through the command line? does it need to be `rake password:validate_strength[password]` or `rake password:validate_strength["password"]`?

Reply:
without quotes

Reply:
how would someone use this? what's driving this change, and what's the key benefit? I don't think I understand it from what's written here